File: property/src/main/java/com/wujia/intellect/terminal/property/mvp/adapter/WyOtherTelAdapter.java
package com.wujia.intellect.terminal.property.mvp.adapter;

import android.content.Context;
import android.view.View;

import com.wujia.intellect.terminal.property.R;
import com.wujia.intellect.terminal.property.mvp.data.WyChildBean;
import com.wujia.lib_common.base.baseadapter.CommonAdapter;
import com.wujia.lib_common.base.baseadapter.base.ViewHolder;

import java.util.List;

/**
 * Author: created by shenbingkai on 2018/12/11 11 24
 * Email:  shenbingkai@gamil.com
 * Description:
 */
public class WyOtherTelAdapter extends CommonAdapter<WyChildBean> {
    public WyOtherTelAdapter(Context context, List<WyChildBean> datas) {
        super(context, R.layout.item_propery_other_tel, datas);
    }

    @Override
    protected void convert(final ViewHolder holder, WyChildBean item, int pos) {

//        holder.setText(R.id.scene_in_mode_tv,item.title);

        if (!isLast(pos)) {
            holder.getView(R.id.line1).setVisibility(View.VISIBLE);
        } else {
            holder.getView(R.id.line1).setVisibility(View.INVISIBLE);
        }
    }

}

File: property/src/main/java/com/jingxi/smartlife/pad/property/mvp/adapter/WyOtherTelAdapter.kt
package com.jingxi.smartlife.pad.property.mvp.adapter

import android.content.Context
import android.view.View

import com.jingxi.smartlife.pad.property.R
import com.jingxi.smartlife.pad.property.mvp.data.WyChildBean
import com.wujia.lib_common.base.baseadapter.CommonAdapter
import com.wujia.lib_common.base.baseadapter.base.ViewHolder

/**
 * Author: created by shenbingkai on 2018/12/11 11 24
 * Email:  shenbingkai@gamil.com
 * Description:
 */
class WyOtherTelAdapter(context: Context, datas: List<WyChildBean>) : CommonAdapter<WyChildBean>(context, R.layout.item_propery_other_tel, datas) {

    override fun convert(holder: ViewHolder, item: WyChildBean, pos: Int) {

        //        holder.setText(R.id.scene_in_mode_tv,item.title);

        if (!isLast(pos)) {
            holder.getView<View>(R.id.line1).visibility = View.VISIBLE
        } else {
            holder.getView<View>(R.id.line1).visibility = View.INVISIBLE
        }
    }

}
